Aptabase.uplugin

```diff
@@ -1,7 +1,7 @@
 ﻿{
 	"FileVersion": 3,
-	"VersionName": "0.1.1",
-	"EngineVersion": "5.2",
+	"VersionName": "0.0.1",
+	"EngineVersion": "5.3",
```

Comment:
does this define the min engine version?

Reply:
When submitting to the marketplace, we will need to have 3 different zips for 3 different engine versions and each zip needs to match. (e.g.: the 5.2 zip needs to have `"EngineVersion": "5.2"`).

When people use it locally, it doesn't matter. They will get a pop-up warning if they open it with a non-matching version, but it will work as long as they compile it.